micro-rdk-server/esp32/main.rs

```diff
@@ -152,7 +153,7 @@ mod esp32 {
         };
         #[cfg(not(feature = "qemu"))]
         let mut server = { builder.build(Esp32H2Connector::default(), Executor::new(), mdns) };
-        server.run_forever();
+        server.run_forever::<Esp32RpcHeapAllocation>();
```

Comment:
I don't think we should add a Generic here

Reply:
should it be an input parameter to `run_forever`?